linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_mac_modification/ansible/shared.yml

```diff
@@ -4,35 +4,52 @@
 # complexity = low
 # disruption = low
 
+- name: detect if rule does not already exist in /etc/audit/rules.d/*
```

Comment:
Watch out for the capitalization of the task names.
And I would change `detect` to `Check`.

Reply:
Same for the task for audit.rules file.